pkgs/tools/networking/oneshot/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "oneshot";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "raphaelreyna";
+    repo = "oneshot";
```

Comment:
Optional ;):

```suggestion
    repo = pname;
```

Reply:
Now we are in bikeshedding territory. ;)
I usually don't do this, because the repository name and the package name are likely to change independently, if at all. So I think I'll keep this little redundancy, if you do not mind.

Reply:
Don't mind.